File: Objective-C/Objective-C/MyFrameLoadDelegate.m
//
//  MyFrameLoadDelegate.m
//  Objective-C
//
//  Copyright (c) 2015 DesktopJavaScript. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "MyFrameLoadDelegate.h"
#import "MyNativeBridge.h"

@interface MyFrameLoadDelegate ()

@end

@implementation MyFrameLoadDelegate

-(void)webView:(WebView *)sender didCreateJavaScriptContext:(JSContext *)context forFrame:(WebFrame *)frame {
    MyNativeBridge *myNativeBridge = [[MyNativeBridge alloc] init];
    context[@"nativeBridge"] = myNativeBridge;
}

@end

File: RemObjects-C#/MyFrameLoadDelegate.cs
using Foundation;
using JavaScriptCore;
using WebKit;

namespace RemObjectsC {

	public class MyFrameLoadDelegate: IWebFrameLoadDelegate {
	
		public void webView(WebView webView) didCreateJavaScriptContext(JSContext context) forFrame(WebFrame frame) {
			MyNativeBridge myNativeBridge = new MyNativeBridge();
			context["nativeBridge"] = myNativeBridge;
		}
	}
}
